backend/src/test/java/gov/cdc/usds/simplereport/service/TestOrderServiceTest.java

```diff
@@ -39,6 +42,22 @@
   @Autowired private PersonService _personService;
   @Autowired private TestDataFactory _dataFactory;
 
+  private static final PersonName AMOS = new PersonName("Amos", null, "Quint", null);
```

Comment:
More duplicate code. There is a factory class used by tests, but it doesn't need to be part of this PR, IMO.

Reply:
Do we have a java faker library on the backend? It'd be helpful to just create a number of random patients without caring about what their names are. Something like this for the factory class:

```java
public List<Person> createMinimalPeople(number: Int, Organization org, Facility fac) {
  // Faker magic here that generates some people with random names
}
```